### ISSUE TITLE
NoMethodError : private method `to_notation' called for #<Naether::Notation:0x5972a2aa>

### ISSUE THREAD

**Issue by r6p** [mguymon/lock_jar#29]
I am getting the following error w/ lock_jar 0.12.3 and naether 0.14.1 on jruby-9000 pre2. Any ideas why?

````
NoMethodError : private method `to_notation' called for #<Naether::Notation:0x5972a2aa>
/Users/r6p/.rvm/gems/jruby-9.0.0.0.pre2/gems/lock_jar-0.12.3/lib/lock_jar/domain/artifact.rb:46:in `initialize'
/Users/r6p/.rvm/gems/jruby-9.0.0.0.pre2/gems/lock_jar-0.12.3/lib/lock_jar/domain/dsl.rb:85:in `jar'
/Users/r6p/Developer/myproj/Buildfile:25:in `block in (root)'
/Users/r6p/.rvm/gems/jruby-9.0.0.0.pre2/gems/lock_jar-0.12.3/lib/lock_jar/domain/dsl.rb:148:in `block in group'

````

````
cmd: gem list naether

*** LOCAL GEMS ***

naether (0.14.1 java)
````

**Comment by mguymon** [comment]
I have not tested LockJar on JRuby 9k, I give it a whirl.

**Comment by mguymon** [comment]
Bug in Naether, method was being defined in the initializer. Easy enought to fix, https://github.com/mguymon/naether/commit/3f21c36724ccb7eeb366a93b8a5406711194808a

I will push a new gem shortly.

**Comment by mguymon** [comment]
released lock_jar 0.12.4
